curses.c

```diff
@@ -518,9 +518,6 @@ void mtr_curses_init() {
 
 	/* Initialize block_map. */
 	block_split = (NUM_FACTORS - 2) / 2;
-	if (block_split > 9) {
```

Comment:
The original (with the constant probably unnecessary) meant to convey that > 9 is not allowed for the following code. Maybe add a comment to this effect?